tests/locations/test_package.py

```diff
@@ -1370,6 +1370,91 @@ def test_move_uncompressed_aip(self):
         assert pkg.current_location == dst_location
 
 
+@pytest.fixture
+@pytest.mark.django_db
+def space(tmp_path):
+    space_dir = tmp_path / "space"
+    space_dir.mkdir()
+
+    staging_dir = tmp_path / "staging"
+    staging_dir.mkdir()
+
+    space = models.Space.objects.create(
+        access_protocol=models.Space.LOCAL_FILESYSTEM,
+        path=space_dir,
+        staging_path=staging_dir,
+    )
+    models.LocalFilesystem.objects.create(space=space)
+    return space
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def location(space):
+    aipstore = models.Location.objects.create(
+        space=space,
+        relative_path="fs-aips",
+        purpose="AS",
+    )
+    return aipstore
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def internal_location(space):
+    return models.Location.objects.create(
+        space=space, purpose=models.Location.STORAGE_SERVICE_INTERNAL, relative_path=""
+    )
+
+
+@pytest.fixture
+def bag_fixture(tmp_path):
+    tmp_dir = tmp_path / "dir"
+    tmp_dir.mkdir()
+    src = os.path.join(FIXTURES_DIR, "working_bag.zip")
+    dst = os.path.join(tmp_dir, "")
+    return shutil.copy(src, dst)
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def package(location, bag_fixture):
+    return models.Package.objects.create(
+        current_location=location,
+        current_path=os.path.join(location.full_path, bag_fixture),
+        package_type="AIP",
+        status="Uploaded",
+    )
+
+
+@pytest.mark.django_db
+@mock.patch(
+    "common.utils.generate_checksum",
+    return_value=mock.Mock(
+        **{
+            "hexdigest.return_value": "098f6bcd4621d373cade4e832627b4f9",
+        }
+    ),
+)
+def test_get_fixity_check_report_send_signals(
+    generate_checksum, package, internal_location
+):
+    """
+    Verifies scenarios of fixity failure signals
+    """
```

Comment:
The docstring is too vague: which scenarios is referring to? how many are they? And the name of the test seems to suggest it's testing the "happy path" which I don't think it is.

I'd recommend you to apply the test naming conventions suggested during https://github.com/artefactual/archivematica/pull/1932. I think that if you name the test clearly using that framework you don't even need the docstring.